src/_repobee/command/issues.py

```diff
@@ -166,17 +166,10 @@ def _log_repo_issues(
                 color
             )  # color takes character space
 
-            id_ = "{}{}/#{}:".format(color, repo.name, issue.number).ljust(
+            id_ = f"{color}{repo.name}/#{issue.number}:".ljust(
                 adjusted_alignment
             )
-            out = "{}{}{}{}created {!s} by {}".format(
-                id_,
-                issue.title,
-                style.RESET,
-                " ",
```

Comment:
Haha, this has to be one of my better moments: putting a space as the argument to a format string. /facepalm